### ISSUE TITLE
Incomplete information on "Disabling GPU" section "SSDT Method"

### ISSUE THREAD

**Issue by ferdinandwinters** [dortania/bugtracker#297]
As the title says, [here is the link to the page](https://dortania.github.io/OpenCore-Install-Guide/extras/spoof.html).

The section, "SSDT Method", contains the text:

> There are many ways to find the path but generally, the easiest way is to get into Device Manager under windows and find the PCI path.
>
> Example of device path for \_SB.PCI0.PEG0.PEGP:

It should say something like,

> The best way to find the path is to open Device Manager, find the GPU you wish to disable and open its properties. Under the "Details" tab you are looking for the property "BIOS device name"

Attached image of RTX 3060 Ti
![image](https://user-images.githubusercontent.com/25633642/204123684-51587d29-d8e6-4100-a53c-95af2767e761.png)

**Comment by abenraj** [comment]
Don't really see anything incomplete here. I believe the phrasing here is simply intended towards a more "in general" context; basically implying that PCIe paths can be found via familiar routes (Linux/Shell commands etc) but is _generally_ considered easier to find via Windows for those who use Windows as their main OS.

**Comment by ferdinandwinters** [comment]
I agree with you. I do think that there needs to be some more expanded information though. I have used the guide several times in the past and for once I had to search for something outside of the scope of the guide and found it quite difficult to find the result.

Searches on Google for "PCI path windows device manager" and similar yielded insufficient results, I only found the "PCI path" by going down the list item by item until I saw a string that looked similar to the example, and even then, Windows doesn't even call that field a "path" or mention "PCI" anywhere in it.

It's just my experience using the guide. I feel that the information in that section can be expanded.
